api/src/pcapi/core/providers/repository.py

```diff
@@ -5,11 +5,13 @@
 from flask_sqlalchemy import BaseQuery
 
 from pcapi.core.offerers.models import Venue
+import pcapi.core.offers.models as offers_models
 from pcapi.models import db
 
 from . import constants
 from . import exceptions
 from . import models
+from ..categories import subcategories
```

Comment:
```suggestion
from pcapi.core.categories import subcategories
```
(en le remontant dans le 2e bloc d'imports, évidemment)